src/clientsettingsdlg.cpp

```diff
@@ -164,13 +167,13 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
     cbxROutChan->setAccessibleName ( tr ( "Right output channel selection combo box" ) );
 
     // enable OPUS64
-    chbEnableOPUS64->setWhatsThis ( "<b>" + tr ( "Enable Small Network Buffers" ) + ":</b> " +
-                                    tr ( "If enabled, the support for very small network audio packets is activated. Very small "
-                                         "network packets are only actually used if the sound card buffer delay is smaller than " ) +
-                                    QString().setNum ( DOUBLE_SYSTEM_FRAME_SIZE_SAMPLES ) +
-                                    tr ( " samples. The "
-                                         "smaller the network buffers, the lower the audio latency. But at the same time "
-                                         "the network load increases and the probability of audio dropouts also increases." ) );
+    chbEnableOPUS64->setWhatsThis (
+        "<b>" + tr ( "Enable Small Network Buffers" ) + ":</b> " +
+        QString ( tr ( "If enabled, the support for very small network audio packets is activated. Very small "
+                       "network packets are only actually used if the sound card buffer delay is smaller than %1 samples. The "
+                       "smaller the network buffers, the lower the audio latency. But at the same time "
+                       "the network load increases and the probability of audio dropouts also increases." ) )
```

Comment:
...as well as the increase in audio artifacts caused by the codec...

Reply:
Yes, but do we need to add it here? This might get too technical (it already is...)

Reply:
The feature itself is technical in nature.  If the user isn't provided with a proper explanation of the effects, they could wonder why it reduces sound quality.  It does, substantially, particularly on bass-heavy sounds.  Unless the user is _unable_ to play without it, due to latency, it's pretty much being recommended to leave it off now, during WorldJam sound check sessions.

Reply:
```suggestion
                       "the network load and the probability of audio dropouts or sound artefacts increases." ) )
```